### ISSUE TITLE
Beam Blockage Calculation Example - RuntimeError: OGR Error: Unsupported SRS

### ISSUE THREAD

**Issue by eoghanharney** [wradlib/wradlib#628]
Hi all,

Just installed wrablib following the instructions (https://docs.wradlib.org/en/stable/installation.html) and was trying to run the jupyter notebook for the beam blockage example (https://docs.wradlib.org/en/stable/notebooks/beamblockage/wradlib_beamblock.html) but got following error:

```python
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
Cell In[4], line 2
      1 coord = wrl.georef.sweep_centroids(nrays, range_res, nbins, el)
----> 2 coords = wrl.georef.spherical_to_proj(
      3     coord[..., 0], coord[..., 1], coord[..., 2], sitecoords
      4 )
      5 lon = coords[..., 0]
      6 lat = coords[..., 1]

File ~/.local/lib/python3.9/site-packages/wradlib/georef/polar.py:228, in spherical_to_proj(r, phi, theta, sitecoords, proj, re, ke)
    225 if proj is None:
    226     proj = projection.get_default_projection()
--> 228 xyz, rad = spherical_to_xyz(r, phi, theta, sitecoords, re=re, ke=ke, squeeze=True)
    230 # reproject aeqd to destination projection
    231 coords = projection.reproject(xyz, projection_source=rad, projection_target=proj)

File ~/.local/lib/python3.9/site-packages/wradlib/georef/polar.py:106, in spherical_to_xyz(r, phi, theta, sitecoords, re, ke, squeeze, strict_dims)
    104 osr = import_optional("osgeo.osr")
    105 if has_import(osr):
--> 106     rad = projection.proj4_to_osr(projstr)
    107 else:
    108     rad = projstr

File ~/.local/lib/python3.9/site-packages/wradlib/georef/projection.py:206, in proj4_to_osr(proj4str)
    204 proj = osr.SpatialReference()
    205 proj.ImportFromProj4(proj4str)
--> 206 proj.AutoIdentifyEPSG()
    208 if proj.Validate() == ogr.OGRERR_CORRUPT_DATA:
    209     raise ValueError(
    210         "proj4str validates to 'ogr.OGRERR_CORRUPT_DATA'"
    211         "and can't be imported as OSR object"
    212     )

File ~/.conda/envs/wradlib/lib/python3.9/site-packages/osgeo/osr.py:616, in SpatialReference.AutoIdentifyEPSG(self, *args)
    614 def AutoIdentifyEPSG(self, *args):
    615     r"""AutoIdentifyEPSG(SpatialReference self) -> OGRErr"""
--> 616     return _osr.SpatialReference_AutoIdentifyEPSG(self, *args)

RuntimeError: OGR Error: Unsupported SRS

```

#### Using Versions

python=3.9.16
wradlib=1.19.1

Thanks very much!

**Comment by kmuehlbauer** [comment]
@eoghanharney I was confronted with the same issue yesterday and have no permanent solution. The workaround in my case was to install `gdal<3.7`. 

I'm still investigating this.

**Comment by eoghanharney** [comment]
@kmuehlbauer Thanks for your quick reply!

I installed gdal3.6.4 and didn't encounter same issue.

**Comment by kmuehlbauer** [comment]
Thanks for checking, @eoghanharney, much appreciated. I'm not really sure, where or what the problem is. But it has something to do with `AutoIdentifyEPSG` function. I've skimmed the GDAL changelog, but did not find any immediate source. Could also have something to do with how conda is building the lib.

Let's leave this issue open until we find the root cause.

**Comment by kmuehlbauer** [comment]
I'm inclined to just skip `AutoIdentifyEPSG()` for newer GDAL.

**Comment by kmuehlbauer** [comment]
fixed in 1.19.2
